07/23.py

```diff
@@ -1,8 +1,12 @@
 def number_of_keys_with_foo_in_name(my_dict):
     """ return an integer, with the number of keys with the string "foo" in them. """
-    pass
-
-
+    z = []
+    for x in my_dict:
+        b = x.find("foo", 0)
+        if b is not -1:
```

Comment:
you should use == instead of "is" here. this is a nuance I can explain later, but use "==" for numbers.